src/EditorFeatures/VisualBasicTest/LineCommit/CommitWithViewTests.vb

```diff
@@ -1133,5 +1133,191 @@ End Class
                 Assert.Equal(expected.NormalizedValue, testData.Workspace.Documents.Single().GetTextBuffer().CurrentSnapshot.GetText())
             End Using
         End Sub
+
+        <WorkItem(9999, "https://github.com/dotnet/roslyn/issues/48745")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.LineCommit)>
+        Public Sub TestCaseChangeInDoLoop()
+            Using testData = CommitTestData.Create(
+                <Workspace>
+                    <Project Language="Visual Basic" CommonReferences="true">
+                        <Document>
+Class C
+    Sub S()
+        Dim Test = 1
+        Do
+            ' While fixed but Test not
```

Comment:
i don't understand this comment.  why do we not want 'Test' fixed?

Reply:
Comment from original bug report, I need to remove.

Reply:
Left over from issue report, I remove it.